lte/gateway/docker/ghz/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# AGW test docker private repository location
+ARG REPO_LOCATION=agw-test.artifactory.magmacore.org/
```

Comment:
It might look nicer to leave the trailing slash out so that using it looks like `${REPO_LOCATION}/`.  Just a style preference though, feel free to ignore :)